client/lib/wp/sync-handler/index.js

```diff
@@ -197,3 +218,7 @@ export class SyncHandler {
 		return localforage.removeItem( key, fn );
 	}
 }
+
+export const pruneRecordsFrom = () => {
+	cacheInvalidation.pruneRecordsFrom( '2 minutes ago' );
```

Comment:
I'm very confused... why do I see three different values that you're passing into `pruneRecordsFrom`? And why would we ever prune records from "2 minutes ago"?